.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,63 +1,27 @@
 [Short description explaining the high-level reason for the pull request]
 
-## Additions
+### Additions  
 
--
+### Changes  
 
-## Removals
+### Removals 
 
--
+### Testing
 
-## Changes
 
--
+### Checklist
 
-## Testing
+_You may update this checklist before and/or after creating the PR. If you're unsure about any of them, please ask, we're here to help! These items are what we are going to look for before merging your code._
 
-1.
-
-## Screenshots
-
-
-## Notes
-
--
-
-## Todos
-
--
-
-## Checklist
-
-- [ ] PR has an informative and human-readable title
+- [ ] Informative and human-readable title, using the format: `[_pt] PR: <description>`
+- [ ] Links are provided if this PR resolves an issue, or depends on another other PR
+- [ ] If submitting a PR to the `dev` branch (the default branch), you have a descriptive [Feature Branch](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow) name using the format: `dev-<description-of-change>` (ie: `dev-revise-levee-masking`)
```

Comment:
"ie" should be "e.g."

Reply:
@mluck PR is included in the example title on line 16 (**[_pt] PR**). Did you have something else in mind?

- [ ] Informative and human-readable title, using the format: `[_pt] PR: <description>`